client/src/components/archive-save/archive-save.tsx

```diff
@@ -32,6 +32,18 @@ export default function ArchiveSave() {
   };
   const copy = hash && hash.length > 0 ? "Backed up to Swarm" : "Not yet backed up to Swarm.";
   const socialArchiveUrl = (url || '').replace('https://gateway.ethswarm.org/access/', '/archive/');
```

Comment:
@codiak - Instead of using url and hardcoding replacing gateway host, advise to use window.location.origin + '/archive/' + hash instead.

Reply:
Yeah this is a little hacky, really we should just pass the hash through and build the URLs in the display component. Although socialArchiveUrl in particular should be relative.